### ISSUE TITLE
Draggable incorrectly calculates list height inside MUI drawer, causes unexpected styling behavior.

### ISSUE THREAD

**Issue by abarrafo** [atlassian/react-beautiful-dnd#1364]
I have not drilled down exactly what is causing this yet, but is seems that inside of a Material UI (MUI) Drawer component react-beautiful-dnd fails to calculate correct list height for onDrag end action. 

### Expected behavior
When dragging list items, item styling and drag end locations should be correctly calculated. 

### Actual behavior
It seems only the visible list is caclulated resulting in styling getting clipped at the end of the visible list.

https://codesandbox.io/embed/material-demo-3str5 

### Steps to reproduce
See code pen example

### Suggested solution?
None yet . . . 

### What version of `React` are you using?
16.8.6

### What version of `react-beautiful-dnd` are you running?
11.0.4

### What browser are you using?
Chrome  74.0.3729.169

### Demo
![2019-06-17 20 30 57](https://user-images.githubusercontent.com/15097921/59645242-dc356f00-913e-11e9-8486-3797dd13bb8c.gif)

**Comment by moonray** [comment]
Something similar happens for me for a parent container that uses flexbox.

**Comment by alexreardon** [comment]
Can you please check your console and tell me if anything is printed?

**Comment by oliviertassinari** [comment]
The issue can be closed (tested with Material-UI v4.3.3), we worked around the problem with: https://github.com/mui-org/material-ui/pull/16281.

**Comment by oliviertassinari** [comment]
TL;DR problem fixed, I think that we can close the issue.

**Comment by danieldelcore** [comment]
No worries, will close 👍
